client/src/Component/MainTable.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
+import "./MainTableStyle.css";
+
+const MainTable = () => {
+	const [clientData, setClientData] = useState(null);
+	const [error, setError] = useState(null);
+
+	useEffect(() => {
+		fetch("/api/clients")
+			.then((response) => {
+				if (!response.ok) {
+					throw Error("We could not fetch the data for that resource");
+				}
+				return response.json();
+			})
+			.then((data) => {
+				setClientData(data);
+				setError(null);
+			})
+			.catch((err) => {
+				setError(err.message);
+				if (error != null) {
+					return <alert>{err.message}</alert>;
```

Comment:
We shouldn't return JSX in `useEffect`. Instead, our JSX should go at the bottom. Let's move this alert in the main `return` above the table